charts/redpanda/chart_template_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"golang.org/x/tools/txtar"
+	v1 "k8s.io/api/apps/v1"
```

Comment:
nit: For consistency let's import this as `appsv1`

Reply:
ok